lib/completion.js

```diff
@@ -45,8 +45,8 @@ completion.completion = function (opts, cb) {
 }
 
 function completion (args, cb) {
-  if (process.platform === 'win32') {
-    var e = new Error('npm completion not supported on windows')
+  if (process.platform === 'win32' && process.env.MSYSTEM !== 'MINGW32') {
```

Comment:
I have Win7 64-bit and it MSYSTEM in MINGW32. Not sure if this can ever be e.g. MINGW64.